cirq/contrib/acquaintance/permutation.py

```diff
@@ -220,9 +220,11 @@ def update_mapping(mapping: Dict[ops.Qid, LogicalIndex],
         operations: The operations to update according to.
     """
     for op in ops.flatten_op_tree(operations):
-        if (isinstance(op, ops.GateOperation) and
-            isinstance(op.gate, PermutationGate)):
-            op.gate.update_mapping(mapping, op.qubits)
+        # Ignoring type warning about passing an abstract type into Type[...].
+        gate = ops.op_gate_of_type(op, PermutationGate)  # type: ignore
```

Comment:
Is this just a style preference?

Reply:
I can revert it if you prefer. It's just much shorter, and it's why we defined these methods.

Reply:
I don't have a preference. IIRC, this is what I originally wrote, and changed it to avoid the type issues.